ui/src/router/details/components/router_details_header/RouterDetailsPageHeader.js

```diff
@@ -58,6 +58,20 @@ export const RouterDetailsPageHeader = ({ router }) => {
         },
       },
     },
+    {
+      title: "Protocol",
+      description: (
+        <EuiTextColor size="s" color={isRouterUpdating ? "subdued" : "default"}>
```

Comment:
Why is this disabled when the router is updating? Like the endpoint, we can show the protocol as well, right?

Reply:
Oh, this is because the protocol is part of the `config` object which is `undefined` together with `timeout` during updating state

Reply:
Ahh ok. The reason they are not set while the router is updating is because they can change between versions. By this logic, may be we should also hide the router endpoint. The downside to this this, there is no other place that can show the endpoint value (eg: router version details) as it's a property of the router. So I suppose we can leave this as it is for now.